### PR TITLE
new library: added box sketch

### PR DESCRIPTION
**Pull request by IvanReznikov**
Added an opened box sketch

### CODE REVIEW

**Comment by dwelle** [comment]
hey @IvanReznikov, it seems you've included more library items than just the box. Can you remove them?

Also, I was playing with the box, and thought we could improve it: made the flaps none-transparent, and the whole box as well, and the insides darker. It's an aesthetic decision, so up to you:

https://excalidraw.com/?#json=4812707823878144,vXIXkDQlGwf7Ma1-g-FY5Q

image

**Comment by IvanReznikov** [comment]
Nice! I like it! Will resolve the conflicts and update the library. Thanks

File: libraries.json [hunk 1]
```diff
@@ -871,6 +871,20 @@
...
+    "authors": [
+      {
+        "name": "Ivan Reznikov",
+        "url": "https://github.com/IvanReznikov"
+      }
+    ],
+    "source": "IvanReznikov/yellow-box.excalidrawlib",
+    "preview": "IvanReznikov/yellow-box.png",
+    "created": "2021-08-26",
+    "updated": "2021-08-27"
```
Comment (dwelle) [inline libraries.json:886]:
```suggestion
    "created": "2021-09-22",
    "updated": "2021-09-22"
```

**Comment by dwelle** [comment]
Btw, the file seems to be an `.excalidraw` file, not `.excalidrawlib`. You can export the lib file from the library menu:

image

File: libraries.json [hunk 2]
```diff
@@ -871,6 +871,20 @@
...
+    "authors": [
+      {
+        "name": "Ivan Reznikov",
+        "url": "https://github.com/IvanReznikov"
+      }
+    ],
+    "source": "IvanReznikov/yellow-box.excalidrawlib",
+    "preview": "IvanReznikov/yellow-box.png",
+    "created": "2021-09-22",
+    "updated": "2021-09-22"
```
Comment (dwelle) [inline libraries.json:886]:
```suggestion
    "created": "2021-09-24",
    "updated": "2021-09-24"
```

**Comment by dwelle** [comment]
fixed and shipped

**Pull request merged**